test/apiv2/01-basic.at

```diff
@@ -1,3 +1,4 @@
+# shellcheck shell=sh
```

Comment:
Please leave the `-*-` line first: this is needed by [emacs](https://www.gnu.org/software/emacs/manual/html_node/emacs/Choosing-Modes.html) to determine syntax highlighting. If this new tool supports it, you could do:
```
# -*- sh -*- shellcheck shell=sh
```

Reply:
Yeah, I felt that it needs to be at the first line. Unfortunately, your suggested solution would not work either. Shellcheck requires that its directives are at the beginning of the line.

But I think it would be helpful to add support for emacs filetypes to `differential-shellcheck`:
- https://github.com/redhat-plumbers-in-action/differential-shellcheck/issues/101

The solution could look something like this:

```sh
# -*- sh -*-
# shellcheck shell=sh
#
# Tests for more image-related endpoints
#
```

Reply:
I had added support for emacs file type specifications, so `differential-shellcheck` now can detect files with `# -*- sh -*-`. So there is no need for adding `# shellcheck shell=sh`.

But it might still be valuable to add a ShellCheck directive to specify if the files are supposed to be POSIX shell or bash.